docs/getting-started.md

```diff
@@ -2,7 +2,7 @@
 title: Getting Started
 ---
 
-The easiest way to get started with Selflow is to use the [Selflow CLI](./ecosystem/cli) and run the [Selflow Daemon](./ecosystem/selflow-daemon). You can follow [this documentation](./ecosystem/cli#installation).
+The easiest way to get started with Selflow is to use the [Selflow CLI](./ecosystem/cli/index.mdx) and run the [Selflow Daemon](./ecosystem/selflow-daemon). You can follow [this documentation](./ecosystem/cli#installation).
```

Comment:
1. the link of `selflow-cli` also exist on line line `29` of this file.
2. there is a typo on line 29 (suggestion from LanguageTool plugin):
<img width="887" alt="image" src="https://github.com/selflow/selflow/assets/134276360/18627615-efbf-4872-ba12-3d4146e0f635">